frontend/src/components/common/NotificationStack.vue

```diff
@@ -3,15 +3,15 @@ import { notifications } from '@/store/NotificationState';
 </script>
 
 <template>
-    <div v-for="(value, key) in notifications">
+    <template v-for="(value, key) in notifications" :key="key">
         <NotificationMessage
             v-if="value.msg !== ''"
             :severity="key.toString()"
             :msg-text="value.msg"
             :hasFullMsg="value.fullMsg !== ''"
             class="notification-stack"
         />
-    </div>
+    </template>
```

Comment:
The div to template change was made to avoid rendering empty elements in the DOM.